src/Npgsql/Internal/Converters/Primitive/TextConverters.cs

```diff
@@ -166,13 +166,13 @@ protected override char ReadCore(PgReader reader)
 
     public override Size GetSize(SizeContext context, char value, ref object? writeState)
     {
-        Span<char> spanValue = stackalloc char[] { value };
+        Span<char> spanValue = [value];
```

Comment:
Just in case, but does it still allocate on stack?

Reply:
Good question... [These docs](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-12.0/collection-expressions#ref-safety) (and some others on the page) seem to indicate that yeah, it's stack-allocated...

Reply:
[This post](https://medium.com/@tauraigombera/pythonians-will-now-love-c-how-c-is-embracing-the-python-way-with-collection-literals-c7e0dad626e4) is more explicit about it...